src/app/shared/asyncapi.service.ts

```diff
@@ -123,6 +123,8 @@ export class AsyncApiService {
     private mapOperation(operationName: string, message: Message, bindings?: any): Operation {
         return {
             type: this.getProtocol(bindings) + operationName,
```

Comment:
As the `type` got split, is it possible to remove the field?

Reply:
yep, gone now.